src/lib/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestEditor/BorrowingRequestForm/BorrowingRequestForm.js

```diff
@@ -15,7 +15,7 @@ import { Payment } from './Payment';
 
 const submitSerializer = values => {
   const submitValues = { ...values };
-  submitValues.library_pid = values.library.pid;
+  submitValues.library_pid = values.library.id;
```

Comment:
Maybe better:
```suggestion
  submitValues.library_pid = values.library.metadata.pid;
```
but you might have to check if it is set? (when creating a new one)

Reply:
I am not sure but I think this `id` is coming from here https://github.com/inveniosoftware/react-invenio-app-ils/blob/70ba2afcbcb433f3cb416070c06c50bd6c170483/src/lib/modules/ESSelector/serializer.js#L108 so I don't think I can change it.